crates/turborepo-ui/src/tui/pane.rs

```diff
@@ -169,7 +168,6 @@ impl<W> TerminalOutput<W> {
             stdin,
             rows,
             cols,
-            has_been_persisted: false,
```

Comment:
This was only needed when we would persist tasks while the TUI was still running. We were only persisting via the `render_remaining` so this was never `true` from a previous call to `persist_screen`